app/views/foreman_openscap/policy_mailer/_dashboard.erb

```diff
@@ -0,0 +1,24 @@
+<table style="border: 1px solid #cccccc; padding: 5px; background-color: #ffffff; border-collapse: collapse; text-align: center; width: 100%">
+  <tbody>
+    <tr>
+      <td style="font-size: 125%; width: 33%; padding: 5px; border: 1px solid #ccc;">
+        <%= (n_('<b style="color: #6bb5df;">%{changed_hosts}</b>
+                <p style="font-weight: bold; margin: 5px; padding: 0px;">Changed</p>',
+                '<b style="color: #6bb5df;">%{changed_hosts}</b>
+                <p style="font-weight: bold; margin: 5px; padding: 0px;">Changed</p>', changed_hosts) % {:changed_hosts => changed_hosts.length}).html_safe %>
+      </td>
+      <td style="font-size: 125%; width: 33%; padding: 5px; border: 1px solid #ccc;">
```

Comment:
Instead of repeating the html style use it in a variable.
See [style in variable](https://github.com/theforeman/foreman/blob/develop/app/views/host_mailer/_active_hosts.html.erb#L1) and [usage](https://github.com/theforeman/foreman/blob/develop/app/views/host_mailer/_active_hosts.html.erb#L4)